.travis.yml

```diff
@@ -12,8 +12,14 @@ go:
   - tip
 
 matrix:
+  include:
+    - go: "1.12"
+      env: GO111MODULE=on
+    - go: tip
+      env: GO111MODULE=on
```

Comment:
can we add a check in the matrix for older versions not using `GO111MODULE=on` (and using `dep ensure`) until we reach go 1.13, where gomodules will be standard? When that happens we will remove all the older dep stuff including the `Gopkg.toml/lock`

Reply:
with this setup it runs 4 builds. 2 without the `GO111MODULE` and 2 with it on.

Reply:
Ok sounds good, this is just me not being familiar with jenkins then.